.npmrc

```diff
@@ -0,0 +1 @@
+ignore-workspace-root-check=true
```

Comment:
Why do we need config for `npm` and not `pnpm`?

Reply:
Because there is no `.pnpmrc` file, we need to create `.npmrc` if we want to change CLI behavior.

[Documentation](https://pnpm.io/npmrc)